homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -729,6 +730,13 @@ def _build_entity_state_dict(entity: State) -> dict[str, Any]:
     return data
 
 
+def _hass_to_hue_state(entity: State) -> bool:
+    if entity.domain == cover.DOMAIN:
```

Comment:
Let's make this a dict lookup by domain with a fallback to OFF

Reply:
I've kept the method but used a dict lookup. I can see the check becoming more complex over time, e.g. when we add multiple off states for a domain (media player?)